src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -20,8 +20,7 @@ export class FeederPodcastModel extends BaseModel {
   authorEmail: string = '';
 
   VALIDATORS = {
-    link: [REQUIRED(), URL('Not a valid URL')],
-    newFeedUrl: [URL('Not a valid URL')]
```

Comment:
I think that we still need the URL validators even with the magic. (The Wysiwyg editor doesn't have good validation on the URL, which could be added in a separate ticket.) If we don't keep the validation here though, it will allow creation of urls such as "http://s". Even though Ryan says it's "not entirely accurate" other URL validation I've worked with is incredibly loose, mostly just requiring a period with valid characters something before and after. So what we have is better than not having it I think, might have to fudge with the 'http|https' part of the test though depending on what string it is validating.